src/main/scala/org/broadinstitute/dsde/firecloud/model/Entity.scala

```diff
@@ -11,12 +11,12 @@ import scala.annotation.meta.field
 @ApiModel(value = "Entity")
 case class Entity  (
                               @(ApiModelProperty@field)(required = true, value = "The namespace the entity belongs to")
-                              wsNamespace: Option[String] = None,
+                              wsNamespace: String,
                               @(ApiModelProperty@field)(required = true, value = "The name of the workspace the entity belongs to")
-                              wsName: Option[String] = None,
+                              wsName: String,
                               @(ApiModelProperty@field)(required = true, value = "The type of the entity")
-                              entityType: Option[String] = None,
+                              entityType: String,
                               @(ApiModelProperty@field)(required = true, value = "The name of the entity")
-                              entityName: Option[String] = None,
+                              entityName: String,
                               @(ApiModelProperty@field)(required = true, value = "The attributes of the entity")
-                              attributes: Option[Map[String, String]] = None)
\ No newline at end of file
+                              attributes: Map[String, String])
```

Comment:
I de-Optionized these because it doesn't really make sense, at least from the Rawls perspective, to have an entity without e.g. an entityType or entityName. I've noticed that all of the model classes seem to be fully Optioned, though, so maybe there's a deeper reason for it?